bulk-revoke-tokens-script/README.md

```diff
@@ -0,0 +1,25 @@
+# bulk-revoke-yammer-token
+This is a powershell script which will invoke Yammer public API to bulk revoke Yammer legacy tokens.
+
+## Usage
+```
+Bulk-Revoke-Tokens.ps1 [-CsvPath] <string> [-BearerToken] <string> [-Is3rdPartyAppOnly] <string> [<CommonParameters>]
```

Comment:
What does "[<CommonParameters>]" refer to?

Reply:
This usage was coming from powershell [Get-Help](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/get-help?view=powershell-6)